### PR TITLE
Feature/upgrade go 1.15.6

### PR DESCRIPTION
**Pull request by Skeeve**
Hi Denys.
Can you please have a look?

I (tried to) upgrade the SDK to go version 1.15.6.

Unfortunately now go-sqlite cannot be compiled anymore.

I get

```text
# runtime/cgo
cc1: warnings being treated as errors
_cgo_export.c:6: warning: ignoring #pragma GCC diagnostic
_cgo_export.c:7: warning: ignoring #pragma GCC diagnostic
_cgo_export.c:8: warning: ignoring #pragma GCC diagnostic
```

And have no clue yet what this means. If you leave out the "go get …" in the Dockerfile, the sdk seems to build.

The only change I did was to change the go-pb.patch to be applied to exec.go instead of build.go now.

### CODE REVIEW

**Comment by Skeeve** [comment]
**Update:** According to https://github.com/mattn/go-sqlite3/issues/907 it seems the version 4.1.2 of arm-none-linux-gnueabi-gcc is outdated fpr go 1.12 and onwards.

I think that version is installed in the pbsdk, right? I haven't found a way yet to upgrade to a newer arm-none-linux-gnueabi-gcc let alone find a newer version of it :(

**Comment by dennwc** [comment]
@Skeeve I found a workaround for it to support the latest Go, but it's in a private branch. I need to rebase it onto your latest changes.

**Comment by Skeeve** [comment]
I think I found a solution, using gcc-arm-linux-gnueabi.

I could compile the sqlitetst and run it successfully on my Touch HD in pbterm.

**Comment by dennwc** [comment]
That's awesome! :tada:  I need to check on my device as well to be sure. But I think it will succeed.

File: Dockerfile [hunk 1]
```diff
@@ -33,12 +33,12 @@ CMD ["build"]
 
 ADD ./* /gopath/src/github.com/dennwc/inkview/
 
-ARG CC=arm-none-linux-gnueabi-gcc
+ARG CC=arm-linux-gnueabi-gcc
 ARG GOOS=linux
 ARG GOARCH=arm
-ARG GOARM=5
+ARG GOARM=7
```
Comment (dennwc) [inline Dockerfile:39]: This is a bit I'm worried about. My device might be older than this. We'll see.

Comment (Skeeve) [reply Dockerfile:39]: Try in pbterm

```bash
cat /proc/cpuinfo
```

For HD3 it displays ARMv7

Comment (Skeeve) [reply Dockerfile:39]: Just tested… Works with `GOARM=5` as well.

Comment (dennwc) [reply Dockerfile:39]: Yep, you were right, it's ARMv7. Issue solved then, I guess :)

**Review by dennwc — APPROVED** [review]

**Pull request merged**